# Lab | Inferential statistics

## It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?
* Set up the hypothesis test.
* Write down all the steps followed for setting up the test.
* Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

H0: The group is significantly different from the regular population

In [14]:
import math as m
import pandas as pd

In [8]:
μ = 120
n = 100
sysBP = 130.1
std = 21.21
alpha = 1.96

In [9]:
z = (sysBP - μ) / (std/m.sqrt(n))
z

4.761904761904759

The z score for our sample data 4.76 is bigger than the alpha 1.96, therefore we reject the null hypothesis.